rcl_action/CMakeLists.txt

```diff
@@ -69,9 +70,22 @@ install(TARGETS ${PROJECT_NAME}
 if(BUILD_TESTING)
   find_package(ament_cmake_gtest REQUIRED)
   find_package(ament_lint_auto REQUIRED)
+  find_package(test_msgs REQUIRED)
   ament_lint_auto_find_test_dependencies()
   ament_find_gtest()
   # Gtests
+  ament_add_gtest(test_action_client
+    test/rcl_action/test_action_client.cpp
+  )
```

Comment:
This test is only covering the default rmw impl. It would be good if the test would be invoked for every rmw impl.